bril-rs/brillvm/src/llvm.rs

```diff
@@ -1294,28 +1294,29 @@ pub fn create_module_from_program<'a>(
                     }
                 });
 
-                (llvm_func, instrs, block, heap)
+                (llvm_func, instrs, block, heap, return_type)
```

Comment:
could preserve original function here instead of instrs and return_type
That, or do a zip later with the original functions and the llvm ones